.storybook/main.ts

```diff
@@ -1,13 +1,13 @@
 import type { StorybookConfig } from "@storybook/react-webpack5";
 const config: StorybookConfig = {
-  stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
+  stories: ["../src/**/*.stories.@(js|jsx|ts|tsx)"],
   addons: [
     "@storybook/addon-links",
     "@storybook/addon-essentials",
     "@storybook/addon-interactions",
-    "@storybook/addon-docs",
     "@storybook/addon-a11y",
-    "@storybook/addon-styling",
+    // "@storybook/addon-docs",
+    // "@storybook/addon-styling",
```

Comment:
@Develliot Why aren't these needed anymore?

Reply:
I styled inside story content with providers, the theme switcher was misleading because it would only switch theme in default stories 

addon-docs you get for free anyway, i saw a warning about it loading twice